Kros.KORM/tests/Kros.KORM.UnitTests/Query/QueryShould.cs

```diff
@@ -33,6 +33,15 @@ public void CreateExpressionFromSql()
             expression.Sql.Should().Be("Select * from TPerson");
         }
 
+        [Fact]
+        public void CreateExpressionFromSqlInterpolation()
+        {
+            var query = CreateQuery();
+            var expression = (query.Sql($"SELECT * FROM TPerson").Expression as SqlExpression);
```

Comment:
Make sure that the query string contains correctly defined names for the parameters. 

```
var expression = (query.Sql($"SELECT * FROM TPerson WHERE Id = {11}").Expression as SqlExpression);
expression.Sql.Should().Be("SELECT * FROM TPerson WHERE Id = @0");
```